games-strategy/ufo-ai/ufo-ai-2.5.ebuild

```diff
@@ -0,0 +1,128 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+inherit eutils flag-o-matic games
+
+MY_P=${P/ufo-ai/ufoai}
+
+DESCRIPTION="UFO: Alien Invasion - X-COM inspired strategy game"
+HOMEPAGE="http://ufoai.sourceforge.net/"
+SRC_URI="mirror://sourceforge/ufoai/${MY_P}-source.tar.bz2
+	mirror://sourceforge/ufoai/${MY_P}-data.tar"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86"
+IUSE="debug server editor +client sse profile"
```

Comment:
Thanks for the comments so far.  I have one question here for clarification: would you suggest to drop the `client` USE flag completely --> by default build both server and client, and if `use dedicated` build just the server?